gatling/user-files/utils/create-users.js

```diff
@@ -2,8 +2,8 @@ const
   rp = require('request-promise'),
   host = process.argv[2] || 'localhost';
 
-const doit = (id) => {
-  for (let it = 1; it < 100; ++it, ++id) {  
+const doit = startingId => {
+  for (let id = startingId || 1; id < startingId + 100; ++id) {  
```

Comment:
From that line of code, I guess that `startingId` can be null or undefined?
If so, the `startingId + 100` part results in a `NaN` value, and the for loop is skipped.

Reply:
Can you explain?
I can't see a case where `startingId` can be null or undefined since its the result of a constant (`usersBundleSize`) multiply by another variable. it can be equal to 0 once, but it doesn't matter

Reply:
I thought that `startingId` could be undefined or null, it didn't occured to me that it could always be an integer, and that it could simply be 0.